adapters/rubicon/rubicon.go

```diff
@@ -408,9 +408,21 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			continue
 		}
 
+		siteId, err := rubiconExt.SiteId.Int64()
+		if err != nil {
+			errs = append(errs, err)
+			continue
```

Comment:
This is just for my own understanding, but is there a chance that the error we can get back is bad enough that we wouldn't want to continue execution? 

I see this style of error handling in a lot of places in this file, so I was just wondering.

Reply:
In most cases we would like to continue execution.